wurst/_items/Blowgun.wurst

```diff
@@ -169,11 +158,8 @@ class Blowpipe
             updateTooltip()
 
     function loadAmmo(unit owner, item itm)
-        if not this.loadingEnabled
-            return
-
         int charges = itm.getCharges()
```

Comment:
`int charges = max(itm.getCharges(), 1)`

Reply:
See if `itm.getCharges() or 1` works. I'm guessing Wurst does not support typing like that, but worth a shot.

Reply:
I think the former is clearer